packages/public/server/src/module/Ui/templates/layout/default/about-serlo.phtml

```diff
@@ -0,0 +1,220 @@
+<svg
```

Comment:
Do you think it's worth cleaning the svg a bit, when using it as inline svg (e.g. get rid of a lot of not needed inkscape attributes)?